apps/web/pages/settings/my-account/appearance.tsx

```diff
@@ -74,8 +76,7 @@ const AppearanceView = () => {
     },
   });
 
-  if (isLoading || isLoadingHasTeamPlan)
-    return <SkeletonLoader title={t("appearance")} description={t("appearance_description")} />;
+  if (isLoading) return <SkeletonLoader title={t("appearance")} description={t("appearance_description")} />;
```

Comment:
We stopped using loading status from team plan query? This might let the content being shown for a moment, won't be a good UX.

Reply:
I added it back.